### PR TITLE
LPS-126072 Show on settings dropdown only languages that are available on form builder

### PR DESCRIPTION
**Pull request by brunofariasdeo**
Related Issue: https://issues.liferay.com/browse/LPS-126072

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js [hunk 1]
```diff
@@ -208,6 +213,7 @@ const LocalizableText = ({
 			const newAvailableLocales = translationManager.get(
 				'availableLocales'
 			);
+
 			const newEditingLocale = transformEditingLocale({
```
[the remark this answers is not in the document]
Comment (brunofariasdeo) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js:217]: Carol, acho que eu devo deixar o `const {availableLocales} = {...transformAvailableLocales(...` logo após o `const newEditingLocale`, uma vez que o `availableLocales` chama o `newEditingLocale`

[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js:217]: Se for uma dependência então sem problemas, só deixa uma linha entre eles pra indicar que a ordem importa 😉

**Comment by carolmariaabb** [comment]
ci:test:stop

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**